subprojects/core-api/src/main/java/org/gradle/api/attributes/CompileView.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.attributes;
+
+import org.gradle.api.Incubating;
+import org.gradle.api.Named;
+
+/**
+ * Represents the depth at which module dependencies are exposed during compilation. Differentiates
```

Comment:
I think the first sentence here should be deleted.  At a minimum, I think the word "depth" should be changed, as it only subtracts clarity.